src/pages/en/guides/images.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Images
+description: Learn how to use images in Astro.
+i18nReady: true
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+Astro provides several ways for you to use images on your site, whether they are stored locally inside your project, linked to remotely, or stored in a CMS or CDN!
+
+### In `.astro` files
+
+Astro uses standard HTML `<img>` or `<img />` elements to display images within your `.astro` files. All HTML image attributes are supported.
+
+```astro
+---
+// src/pages/index.astro
+import rocket from './images/rocket.svg';
+---
+<!-- Remote image on another server -->
+<img src="https://astro.build/assets/logo.png" width="25" alt="Astro">
+
+<!-- Local image stored at public/assets/stars.png -->
+<img src="/assets/stars.png" alt="A starry night sky.">
+
+<!-- Local image stored at src/images/rocket.svg -->
+<img src={rocket} alt="A rocketship in space."/>
+```
+
+### In .`md` files
+
+You can use standard Markdown `![]()` syntax or standard HTML `<img>` tags in your `.md` files for images located in your `public/` folder, or remote images on another server.
+
+
+```md
+// src/pages/post-1.md
+
+# My Markdown Page
+
+<!-- Local image stored at public/assets/stars.png -->
+![A starry night sky.](/assets/stars.png)
+<img src="/assets/stars.png" alt="A starry night sky.">
+
+<!-- Remote image on another server -->
+![Astro](https://astro.build/assets/logo.png)
+<img src="https://astro.build/assets/logo.png" width="25" alt="Astro">
+```
+
+### In `.mdx` files
+
+You can use standard Markdown `![]()` syntax or  JSX `<img />` tags in your `.mxd` files. Just like Markdown files, MDX files can display images from your `public/` folder or remote servers. They can also import and use images located in your project's `src` directory, like Astro components.
+
+```mdx
+// src/pages/post-1.md
+
+import rocket from '..images/rocket.svg';
+
+# My MDX Page
+
+// Local image stored at src/images/rocket.svg
+<img src={rocket} alt="A rocketship in space."/>
+
+// Local image stored at public/assets/stars.png
+![A starry night sky.](/assets/stars.png)
+<img src="/assets/stars.png" alt="A starry night sky." />
+
+// Remote image on another server
+![Astro](https://astro.build/assets/logo.png)
+<img src="https://astro.build/assets/logo.png" width="25" alt="Astro" />
+```
+
+### In UI Framework Components
+
+When adding images in a [UI framework component](/en/core-concepts/framework-components/) (e.g React, Svelte), use the image syntax appropriate for that particular component's framework.
+
+## Where to keep images
+
+### `src/`
+Your images kept in `src/` can be imported and used by other components, and will be processed during Astro's build process. These images are imported from a **relative [file path](https://www.codecademy.com/resources/docs/html/file-paths)** and the image's `src` attribute should refer to the import. 
```

Comment:
This sentence sounded better to me without the comma.
```suggestion
Your images kept in `src/` can be imported and used by other components and will be processed during Astro's build process. These images are imported from a **relative [file path](https://www.codecademy.com/resources/docs/html/file-paths)** and the image's `src` attribute should refer to the import. 
```
Alternatively, maybe we could simplify the sentence? What makes the reading flow weird to me is using `and` two times in the same sentence. I think we don't need to say "can be imported and used", I would infer that I need to import it somehow and the next sentence talks exactly about importing so I think it's safe to just remove it.

```suggestion
Your images kept in `src/` can be used by other components and will be processed during Astro's build process. These images are imported from a **relative [file path](https://www.codecademy.com/resources/docs/html/file-paths)** and the image's `src` attribute should refer to the import. 
```

Reply:
Great! Both you and @hippotastic called out this paragraph, so excellent, I will take another pass at this one with both your feedback!